crates/store/src/state.rs

```diff
@@ -1021,22 +1031,9 @@ impl State {
 #[instrument(level = "debug", target = COMPONENT, skip_all)]
 async fn load_nullifier_tree(db: &mut Db) -> Result<NullifierTree, StateInitializationError> {
     let nullifiers = db.select_all_nullifiers().await?;
-    let len = nullifiers.len();
-
-    let now = Instant::now();
-    let nullifier_tree = NullifierTree::with_entries(
-        nullifiers.into_iter().map(|info| (info.nullifier, info.block_num)),
-    )
-    .map_err(StateInitializationError::FailedToCreateNullifierTree)?;
-    let elapsed = now.elapsed().as_secs();
-
-    info!(
-        num_of_leaves = len,
-        tree_construction = elapsed,
-        COMPONENT,
-        "Loaded nullifier tree"
-    );
-    Ok(nullifier_tree)
+
+    NullifierTree::with_entries(nullifiers.into_iter().map(|info| (info.nullifier, info.block_num)))
+        .map_err(StateInitializationError::FailedToCreateNullifierTree)
```

Comment:
We can simplify this because we already instrument the entire function - right?

If so, should we change the `level` from "debug" to info"? Since this is executed only on startup, I don't think this would create too much "noise".

I'd do the same with `load_mmr` and `load_accounts` (which we should probably rename into `load_account_tree`).